services/ui-waflog-s3-bucket/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
There should be a yarn.lock file checked in.  It's missing.

Reply:
added file